pekko-sample-cluster-client-grpc-scala/project/plugins.sbt

```diff
@@ -1,3 +1,3 @@
-addSbtPlugin("com.lightbend.akka.grpc" % "sbt-akka-grpc" % "0.8.4")
+addSbtPlugin("org.apache.pekko" % "sbt-pekko-grpc" % "0.8.4")
```

Comment:
Are we not publishing the sbt-pekko-grpc plugins yet?

I thought we were, looking at the successful pipeline : https://github.com/apache/incubator-pekko-grpc/actions/runs/4425762836/jobs/7761223926 . However, even with updated grpc version to `0.0.0-15-3d8bff9d-SNAPSHOT`, sbt multi-jvm:test still failed due to the package not being found. Therefore i've reverted back to the akka dependency.

Error message:

```
[error] (update) sbt.librarymanagement.ResolveException: Error downloading org.apache.pekko:sbt-pekko-grpc_2.12_1.0;sbtVersion=1.0;scalaVersion=2.12:0.0.0-15-3d8bff9d-SNAPSHOT
[error]   Not found
[error]   Not found
[error]   not found: https://repo1.maven.org/maven2/org/apache/pekko/sbt-pekko-grpc_2.12_1.0_2.12_1.0/0.0.0-15-3d8bff9d-SNAPSHOT/sbt-pekko-grpc_2.12_1.0-0.0.0-15-3d8bff9d-SNAPSHOT.pom
[error]   not found: /home/developer/.ivy2/localorg.apache.pekko/sbt-pekko-grpc_2.12_1.0/scala_2.12/sbt_1.0/0.0.0-15-3d8bff9d-SNAPSHOT/ivys/ivy.xml
[error]   not found: https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/org.apache.pekko/sbt-pekko-grpc_2.12_1.0/scala_2.12/sbt_1.0/0.0.0-15-3d8bff9d-SNAPSHOT/ivys/ivy.xml
[error]   not found: https://repo.typesafe.com/typesafe/ivy-releases/org.apache.pekko/sbt-pekko-grpc_2.12_1.0/scala_2.12/sbt_1.0/0.0.0-15-3d8bff9d-SNAPSHOT/ivys/ivy.xml
```

Reply:
I can see the referenced files in https://repository.apache.org/content/repositories/snapshots/org/apache/pekko/sbt-pekko-grpc_2.12_1.0/0.0.0-15-3d8bff9d-SNAPSHOT/ but this is not being pulled

Reply:
you seem not to have defined the resolver for repository.apache.org unless you clipped the error messages above - 4 repos searched but not the right one